src/main/scala/org/nbastats/model/HasBasicAvgStats.scala

```diff
@@ -0,0 +1,11 @@
+package org.nbastats.model
+
+trait HasBasicAvgStats {
```

Comment:
i think we can have this trait extend `HasBasicStats`, and provide an implementation for the averages rather than leaving them abstract

Reply:
yes